zanata-server/runapp.sh

```diff
@@ -6,11 +6,16 @@ if [ -n "${ZANATA_VERSION-}" ];then
     ZanataVer=$ZANATA_VERSION
 else
     ## Use the ZANATA_VERSION from Dockerfile, line "ARG ZANATA_VERSION="
-    ZanataVer=$(sed -r -n -e '/^ARG\s+ZANATA_VERSION/ s/^ARG\s+ZANATA_VERSION\s*=\s*(.*)\s*/\1/p' < $ScriptDir/Dockerfile)
+    ZanataVer=latest
```

Comment:
I don't have a problem with this change, but @definite should take a look at it. The right implementation depends on how we expect people to use this script.